tutorials/20_REST_API.ipynb

```diff
@@ -0,0 +1,485 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Haystack with REST API\n",
+    "\n",
+    "- **Level**: Intermediate\n",
+    "- **Time to complete**: 30 minutes\n",
+    "- **Prerequisites**: N/A\n",
```

Comment:
Honestly I would suggest making this tutorial condensed to 'using the Rest API'. Lmk what you think here as I think this might be a bit controversial:
The usual problems we see is simply technical, in terms of just setting up, using and interacting with docker and rest api. So I would argue understanding in detail the pipeline structure and each component isn't the aim here. Imo the things people should come away with is:
- These .yml files are where I define my config
- They are able to edit the file in a way, maybe with some examples (change the model, reader setting etc) and use this as the pipeline that they query with rest api
- For details on each NLP related component and par I would forward them to the relevant docs (e.g. the full set of options for a preprocessor, or reader, or retriever etc)

Adding this comment as a reply here because of the mention of 'NLP concepts' which I think _could_ be added as a prerequisite. But mainly I would say the prerequisites are: basic understanding of the YAML syntax (link to resource), basic understanding of docker.